### PR TITLE
Package generation/generate genesis config

### LINKED ISSUE #15595: create a separete job of package generation pipeline that print out the new genesis config

### PR DESCRIPTION
**Pull request by ghost-not-in-the-shell**
Explain your changes:
prints out the new genesis config during package generation pipeline

Explain how you tested your changes:
*

Checklist:

- [ ] Dependency versions are unchanged
  - Notify Velocity team if dependencies must change in CI
- [ ] Modified the current draft of release notes with details on what is completed or incomplete within this project
- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them

* Closes #15595

### CODE REVIEW

File: buildkite/src/Command/MinaArtifact.dhall [hunk 1]
```diff
@@ -98,7 +98,24 @@ let hardforkPipeline : DebianVersions.DebVersion -> Pipeline.Config.Type =
         , mode = PipelineMode.Type.PackageGeneration
         }
       , steps =
-        [ Command.build
+        [ Command.build Command.Config::{
+            commands = [
+              Cmd.runInDocker Cmd.Docker::{
+                image = "\\\${MINA_DAEMON_DOCKER_IMAGE}"
```
Comment (georgeee) [inline buildkite/src/Command/MinaArtifact.dhall:104]:
```suggestion
                image = "\\\${MINA_DAEMON_DOCKER_IMAGE:-gcr.io/o1labs-192920/mina-daemon@sha256:6da66879aacab050a6955c84347f587e43044987b04b9de4522a942f770cc5e7}"
```

I took this image from [recent HF package nightly build](https://buildkite.com/o-1-labs-2/hardfork-package-generation/builds/270#018f32be-4103-4ae4-9a2a-24c7f11c0b7d). Devnet/mainnet distinction doesn't affect `mina-create-genesis` tool.

**Review by georgeee — APPROVED** [review]

**Review by dkijania — APPROVED** [review]

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Comment by ghost-not-in-the-shell** [comment]
!ci-build-me

**Review by georgeee — APPROVED** [review]

**Pull request merged**